spec.md

```diff
@@ -377,24 +377,31 @@ how Platforms might expose these values to their users.
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Instance. |
-| [update](#input-parameters-object) | object | The schema definition for updating a Service Instance. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Instance. |
+| [update](#json-schema-objects) | object | The schema definition for updating a Service Instance. |
 
 
 ##### Service Binding Object
 
 | Response field | Type | Description |
 | --- | --- | --- |
-| [create](#input-parameters-object) | object | The schema definition for creating a Service Binding. |
+| [create](#json-schema-objects) | object | The schema definition for creating a Service Binding. |
 
 
-##### Input Parameters Object
+##### JSON Schema Objects
+
+Platforms MAY be able to provide a better user experience if Service Brokers can
```

Comment:
I think this "MAY" should be "might" since this isn't really something controlled by the spec - so its non-normative IMO.